flux/clusters/k8s-01/keycloak-db/base/configmap.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
```

Comment:
Not sure if init.sql needed for KK db. I would probably remove this, manually restore db and in case if some actions will require add this configmap back with correct SQL commands.